rethinkdb/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,47 @@
+name: RethinkDB
+
+files:
+  - name: rethinkdb.yaml
+
+    options:
+      - template: init_config
+        options:
+          - template: init_config/default
+
+      - template: instances
+        options:
+          - name: host
+            required: false
```

Comment:
Not required?

Reply:
Depends on the meaning of "required" here: we have a default, so the user doesn't _have_ to set this up, but at the same time if there's no value then we can't connect… I assume `required` refers to the latter case?